Source/Core/Common/CMakeLists.txt

```diff
@@ -152,6 +152,8 @@ add_library(common
   UPnP.cpp
   UPnP.h
   VariantUtil.h
+  VCRuntimeUpdater.cpp
+  VCRuntimeUpdater.h
```

Comment:
I don't know a whole lot about CMake, but doesn't this need to go in an `if (MSVC)` (or rather `if (WIN32)`) block, since it only makes sense on Windows?
At least that would be my guess why everything but Windows is currently failing on CI.

Reply:
Yes, this should be using `if (WIN32)`.